engine/src/main/java/org/hibernate/search/engine/cfg/impl/EngineConfigurationUtils.java

```diff
@@ -17,20 +17,9 @@
 	private EngineConfigurationUtils() {
 	}
 
-	public static ConfigurationPropertySourceExtractor extractorForBackend(Optional<String> backendNameOptional,
-			String defaultBackendName) {
+	public static ConfigurationPropertySourceExtractor extractorForBackend(Optional<String> backendNameOptional) {
 		if ( !backendNameOptional.isPresent() ) {
-			if ( defaultBackendName == null ) {
-				return engineSource -> engineSource.withMask( EngineSettings.Radicals.BACKEND );
```

Comment:
Same for this commit: this looks like https://hibernate.atlassian.net/browse/HSEARCH-4071 material?